cmd/kube-apiserver/app/server.go

```diff
@@ -211,6 +213,8 @@ func (s *APIServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.AdmissionControlConfigFile, "admission-control-config-file", s.AdmissionControlConfigFile, "File with admission control configuration.")
 	fs.StringSliceVar(&s.EtcdServerList, "etcd-servers", s.EtcdServerList, "List of etcd servers to watch (http://ip:port), comma separated. Mutually exclusive with -etcd-config")
 	fs.StringVar(&s.EtcdConfigFile, "etcd-config", s.EtcdConfigFile, "The config file for the etcd client. Mutually exclusive with -etcd-servers.")
+	fs.StringSliceVar(&s.EtcdEventServerList, "etcd-event-servers", s.EtcdEventServerList, "List of etcd servers for events to watch (http://ip:port), comma separated. Mutually exclusive with -etcd-event-config")
```

Comment:
To be clear - it's actually defaulted in the code, which means if neither of this flag is set, we don't overwrite it.

But yet - I'm going to make possible to overwrite storage for any resource.